src/Symfony/Component/ExpressionLanguage/Node/Node.php

```diff
@@ -75,4 +75,27 @@ public function evaluate($functions, $values)
 
         return $results;
     }
+
+    public function dump()
+    {
+        return sprintf('<Dumping a "%s" instance is not supported yet>', get_class($this));
```

Comment:
Wouldn't it be better to just throw an exception here? As the dumped expression is not usable anyway.